lib/js/components/Buttons/IconButton/IconButton.consts.ts

```diff
@@ -21,6 +20,12 @@ export const ICON_BUTTON_COLOR_SCHEMES = {
 } as const;
 
 export const ICON_BUTTON_COLORS = {
-	...COLORS,
-	...BUTTON_COLORS,
+	PRIMARY: 'primary',
+	NEUTRAL: 'neutral',
```

Comment:
after this change story for `IconButton` won't work correctly. I see multiple usages of `BUTTON_COLORS.MINOR` there

Reply:
I'm moving Your comment to IconButton PR.